.rubocop.yml

```diff
@@ -29,10 +31,19 @@ Style/GlobalVars:
 Metrics/BlockLength:
   Exclude:
     - 'spec/**/*.rb'
+    - aws-sdk-rails.gemspec
 
 Metrics/ModuleLength:
   Exclude:
     - 'spec/**/*.rb'
 
 Style/HashSyntax:
   EnforcedShorthandSyntax: never
+
+Style/Documentation:
+  Exclude:
```

Comment:
I think the `# @api private` is applicable for most if not all of these. It should satisfy rubocop and not have them be documented in yard.

Reply:
I'm not changing code not related to this PR